pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -49,6 +49,7 @@ const (
 var supportedTypes = []schema.GroupVersionResource{
 	corev1.SchemeGroupVersion.WithResource("configmaps"),
 	corev1.SchemeGroupVersion.WithResource("secrets"),
+	corev1.SchemeGroupVersion.WithResource("serviceaccounts"),
```

Comment:
I forgot to add this in #2035

Reply:
I feel very sorry for this

Reply:
No, I'm asking not for blame :). And I always appreciate the contributions you made to the project. 
Just want to figure out the situation and how to improve it.